app/src/features/HomeScreen/components/header/index.tsx

```diff
@@ -0,0 +1,56 @@
+import { Button } from "@mui/material";
+import { useDispatch, useSelector } from "react-redux";
+import { logoutCustomer } from "../../../authentication/redux/reducers";
+import { selectUser } from "../../../authentication/redux/selectors";
+import SignIn from "../../../authentication/signIn";
+import SignUp from "../../../authentication/signUp";
+
+type Props = {
+    isUserLoggedIn?: boolean;
+    shouldRenderSignInModal: boolean;
+    shouldRenderSignUpModal: boolean;
+    onSignInModalClose: () => void;
+    onSignUpModalClose: () => void;
+    onSignUpLinkCicked: () => void;
+    openSignInModal: () => void;
+    openSignUpModal: () => void;
+}
+
+const HomeScreenHeader = (
```

Comment:
Zar ne bi header trebalo sam da se brine o renderovanju modala za sign in/sign up umesto da prima preko propova ove stvari?
Kontam da cemo ovaj header prikazivati svuda po sajtu a ne samo na home screen-u pa bi bilo mozda dobro da je nezavisan, tj da moze s minimalno propova da se napravi.

Ili jos bolje da ga renderujemo u _App.tsx_ pre bilo kakvog rutiranja